### PR TITLE
feat: add open graph tags to all pages

### PR DESCRIPTION
**Pull request by Giovanni-Schroevers**

### CODE REVIEW

File: packages/magento-product/components/ProductPageMeta/ProductPageMeta.tsx [hunk 1]
```diff
@@ -16,24 +18,20 @@ export function ProductPageMeta(props: ProductPageMetaFragment) {
...
   return (
     <PageMeta
       title={meta_title ?? name ?? ''}
       metaDescription={meta_description ?? name ?? ''}
       canonical={productLink}
+      openGraphImage={media_gallery?.[0]?.url}
     >
       {sku && (
         <>
           <meta property='type' content='product' key='og-type' />
```
Comment (paales) [inline packages/magento-product/components/ProductPageMeta/ProductPageMeta.tsx:31]: We're using a type here, but later we're using og:type. Is that something we need to move to PageMeta as well?

File: packages/next-ui/PageMeta/PageMeta.tsx [hunk 2]
```diff
@@ -31,6 +31,7 @@ export type PageMetaProps = {
   metaDescription?: string
   metaRobots?: MetaRobotsAll | MetaRobots[]
   children?: React.ReactNode
+  openGraphImage?: string | null
```
Comment (paales) [inline packages/next-ui/PageMeta/PageMeta.tsx:34]: Rename to ogImage?

File: packages/next-ui/PageMeta/PageMeta.tsx [hunk 3]
```diff
@@ -99,10 +107,17 @@ export function PageMeta(props: PageMetaProps) {
...
-        <meta name='description' content={metaDescription.trim()} key='meta-description' />
+        <>
+          <meta name='description' content={metaDescription.trim()} key='meta-description' />
+          <meta property='og:description' content={metaDescription.trim()} />
+        </>
       )}
       <meta name='robots' content={metaRobots.join(',')} key='meta-robots' />
       {canonical && <link rel='canonical' href={canonical} key='canonical' />}
+      <meta property='og:title' content={title.trim()} />
+      <meta property='og:type' content='website' />
```
Comment (paales) [inline packages/next-ui/PageMeta/PageMeta.tsx:118]: See above comment, what is the difference between og:type and type?

**Review by paales — CHANGES_REQUESTED** [review]

**Review by paales — CHANGES_REQUESTED** [review]

File: packages/magento-category/components/CategoryHeroNav/CategoryHeroNav.graphql [hunk 4]
```diff
@@ -3,6 +3,7 @@ fragment CategoryHeroNav on CategoryTree {
   level
   is_anchor
   name
+  image
```
[the remark this answers is not in the document]
Comment (Giovanni-Schroevers) [reply packages/magento-category/components/CategoryHeroNav/CategoryHeroNav.graphql]: This is used in `examples/magento-graphcms/pages/[...url].tsx` but this should have been done in `CategoryMeta`

**Pull request merged**